src/inmanta/config.py

```diff
@@ -114,7 +118,12 @@ def get(
 
         opt = cls.validate_option_request(section, name, default_value)
 
-        val = cfg.get(section, name, fallback=default_value)
+        val = os.environ.get(f"INMANTA_{section.upper()}_{_to_env_name(name)}", default=None)
+        if val:
```

Comment:
An empty string evaluates to False. As such, it would not be possible to set an Inmanta setting to an empty string using an environment variable. We should change this to `val is not None`

Reply:
resolved